.github/actions/aib/action.yml

```diff
@@ -51,13 +51,13 @@ runs:
     - name: Download builder script
       shell: bash
       run: |
-        curl -o /tmp/auto-image-builder.sh "https://gitlab.com/CentOS/automotive/src/automotive-image-builder/-/raw/main/auto-image-builder.sh"
+        curl -o /tmp/auto-image-builder.sh "https://gitlab.com/lrossett/automotive-image-builder/-/raw/script-fix/auto-image-builder.sh?ref_type=heads"
```

Comment:
we should make the AIB variable overridable through an environment variable to sold this